src/types/redis_bloom_chain.cc

```diff
@@ -105,9 +106,14 @@ rocksdb::Status BloomChain::bloomCheck(const Slice &bf_key, const std::string &i
   BlockSplitBloomFilter block_split_bloom_filter;
   block_split_bloom_filter.Init(std::move(bf_data));
 
-  uint64_t h = BlockSplitBloomFilter::Hash(item.data(), item.size());
-  *exist = block_split_bloom_filter.FindHash(h);
-
+  for (size_t i = 0; i < items.size(); ++i) {
+    if (exists->at(i)) {
+      continue;
+    }
+    std::string item = items[i].ToString();
+    uint64_t h = BlockSplitBloomFilter::Hash(item.data(), item.size());
```

Comment:
Why using `ToString` to copying here? Slice also has get `const char*` and length